contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogList.vue

```diff
@@ -301,6 +307,16 @@
 
 <style lang="scss" scoped>
 
+  .catalog-page-wrapper {
+    display: flex;
+    min-height: 100vh;
+  }
+
+  .catalog-main-content {
+    flex: 1;
+    min-height: calc(100vh - 64px);
+  }
```

Comment:
We usually try to avoid using the `vh` unit, as it has some flaky behavior on mobile devices. So let's use 100% instead! If 100% is not enough, then we can take a look at what is going on and try to solve it without the `vh` unit.